### ISSUE TITLE
Unable to re-issue invitation email for previously removed and re-added team member

### ISSUE THREAD

**Issue by jordanh** [ParabolInc/action#545]
## Issue - Bug

This may be working as designed. Not a priority to fix straight away, but I am opening this to discuss and track it.

If I execute the following steps:

1. Create a new team
1. Invite a team member such as `terry@prbl.co`
1. In a separate browser instance, accept the invitation as `terry@prbl.co`
1. Remove the team member `terry@prbl.co`
1. Re-add the team member `tery@prbl.co`

We won't generate and send a new invitation email.

- **Estimated effort:** 2 points ([see CONTRIBUTING.md](https://github.com/ParabolInc/action/blob/master/CONTRIBUTING.md#points-and-sizes))

**Comment by mattkrick** [comment]
heh, i'm actually in that mutation right now working on validation:

```
if (inactiveTeamMembers.length > 0) {
        const inactiveTeamMemberIds = inactiveTeamMembers.map((m) => m.id);
        await r.table('TeamMember')
          .getAll(r.args(inactiveTeamMemberIds), {index: 'id'})
          .update({isNotRemoved: true});
        const inactiveTeamMemberEmails = inactiveTeamMembers.map((m) => m.email);
        const newInvitees = invitees.filter((i) => !inactiveTeamMemberEmails.includes(i.email));
        // TODO send email & maybe pop toast saying that we're only reactiving
        asyncInviteTeam(authToken, teamId, newInvitees);
      } else {
        asyncInviteTeam(authToken, teamId, invitees);
      }
```

it is working as intended, but i agree an email might be nice. after beta, let's discuss writing a "Welcome Back" email template.

**Comment by jordanh** [comment]
This should be fixed now as of `v0.22.0`
